src/WormholeEndpoint.sol

```diff
@@ -8,8 +8,9 @@ import "./interfaces/IWormhole.sol";
 import "./interfaces/IWormholeEndpoint.sol";
 import "./Endpoint.sol";
 import "wormhole-solidity-sdk/libraries/BytesParsing.sol";
+import "./libraries/Pausable.sol";
 
-abstract contract WormholeEndpoint is Endpoint, IWormholeEndpoint, IWormholeReceiver {
+abstract contract WormholeEndpoint is Endpoint, IWormholeEndpoint, IWormholeReceiver, Pausable {
```

Comment:
Should we inherit `Pausable` in the Endpoint contract instead? Pausing an endpoint shouldn't be Wormhole specific if pausing a Manager isn't WH specific right?

Reply:
yea that's fair